core/include/traccc/finding/finding_algorithm.ipp

```diff
@@ -75,9 +75,13 @@ finding_algorithm<stepper_t, navigator_t>::operator()(
 
     // Copy seed to input parameters
     std::vector<bound_track_parameters> in_params;
+    std::vector<unsigned int> n_trks_per_seed;
+    n_trks_per_seed.reserve(seeds.size());
```

Comment:
I think it is easier to do in this way:
`std::vector<unsigned int> n_trks_per_seed(seeds.size(), 0);`

Reply:
`n_trks_per_seed.push_back(0);` will need to be removed together